yb-voyager/cmd/analyzeSchema.go

```diff
@@ -89,6 +89,7 @@ var (
 	amRegex              = regexp.MustCompile(`(?i)CREATE ACCESS METHOD ([a-zA-Z0-9_."]+)`)
 	idxConcRegex         = regexp.MustCompile(`(?i)REINDEX .*CONCURRENTLY ([a-zA-Z0-9_."]+)`)
 	storedRegex          = regexp.MustCompile(`(?i)([a-zA-Z0-9_]+) [a-zA-Z0-9_]+ GENERATED ALWAYS .* STORED`)
+	createTableRegex 	 = regexp.MustCompile(`(?i)CREATE TABLE (IF NOT EXISTS)?([^,]+(?:,[^,]+){0,}, PRIMARY KEY ([^,]+(?:,[^,]+){0,})) PARTITION BY RANGE([^,]+(?:,[^,]+){0,});`)
```

Comment:
Change the name of the variable to refer to the actual part of the regex relating to which the error may arise, for eg. `partitionRegex` (if there are no other regex related to partition, else extend it to be more specific)